libsrc/geos-cbm/setnewmode.s

```diff
@@ -0,0 +1,18 @@
+;
+; Scott Hutter
+;
+; 18.12.18
+
+; void SetNewMode(void);
+
+            .export _SetNewMode
+
+            .include "jumptab.inc"
+            .include "geossym.inc"
+        
+_SetNewMode:
+			lda graphMode
+			eor #$80
+			sta graphMode
+			jsr SetNewMode
```

Comment:
As I mentioned in https://github.com/cc65/cc65/issues/814#issuecomment-447573202 the source code I pointed you at uses tail call elimination in https://github.com/cc65/cc65/blob/master/libsrc/geos-common/graph/bitmapclip.s#L24. Please do so here too (aka replace JSR + RTS with JMP).

Reply:
I tried as you said, if Im understanding correctly:

_SetNewMode:
            lda graphMode
            eor #$80
            sta graphMode
            jmp SetNewMode

causes the program to crash.

Reply:
disregard.,, stupid mistake on my part.  Curious as to why the jmp instead of the rts though, but yes this works fine.

Reply:
> Curious as to why the jmp instead of the rts 

:-) To shave-off unnecessary byte

Reply:
Precisely. You save a precious byte and a few microseconds ;-)